### PR TITLE
40 update readme

### PR DESCRIPTION
**Pull request by b-yap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,102 @@
...
+--rpc-cors all \
+--rpc-external \
+--rpc-methods Unsafe \
+--name amplitude-collator-1 \
+--ws-port 9945 --port 30335 --rpc-port 9935 \
+--chain res/amplitude-spec-raw.json \
+--execution=Native \
+-- \
+--port 30334 \
+--chain kusama-raw.json \
```
Comment (gonzamontiel) [inline README.md:79]: Should we provide a link to this file? Its's not clear needs to get the json from. https://github.com/paritytech/polkadot/blob/master/node/service/chain-specs/kusama.json

File: README.md [hunk 2]
```diff
@@ -1,9 +1,102 @@
...
+```
+./target/release/parachain-collator
+--collator \
+--allow-private-ipv4 \
+--unsafe-ws-external \
+--rpc-cors all \
+--rpc-external \
+--rpc-methods Unsafe \
+--name <INSERT_NAME> \
+--ws-port <WS_PORT> --port <PARA_PORT> --rpc-port <RPC_PORT> \
```
Comment (gonzamontiel) [inline README.md:56]: I think is good to have this knd of variables like WS_PORT. But we also would need to explain what they mean.
Also, idk what `PARA_PORT` means, but `--port` is used as the _peering port_ ,  to communicate with other nodes.

Comment (b-yap) [reply README.md]: I added a table to define them

File: README.md [hunk 3]
```diff
@@ -1,9 +1,102 @@
...
+```
 
-🚧 This is a work in progress. 🚧
+### How to Build
+To build the project, execute
+```
+cargo b --release
+```
+A successful build will create a **target** folder with a sub folder **release**.
+Check for `parachain-collator` in the **release** folder.
```
[the remark this answers is not in the document]
Comment (b-yap) [reply README.md]: removed, :ok_hand:

**Review by gonzamontiel — APPROVED** [review]
lgtm

**Pull request merged**